governator-test-junit/src/main/java/com/netflix/governator/guice/test/TestInjectorCreationMode.java

```diff
@@ -0,0 +1,6 @@
+package com.netflix.governator.guice.test;
+
+public enum TestInjectorCreationMode {
```

Comment:
dont think this enum needs the 'Test' prefix.  There's only one injector in this context, no need to narrow it down to the 'Test' one.